packages/easy-coding-standard/README.md

```diff
@@ -35,6 +35,8 @@ Are you already using another tool?
 composer require symplify/easy-coding-standard --dev
 ```
 
+or download `ecs.phar` from https://github.com/symplify/easy-coding-standard-prefixed/blob/master/ecs.phar
```

Comment:
Link https://github.com/symplify/easy-coding-standard-prefixed here

+ update README of https://github.com/symplify/easy-coding-standard-prefixed to point 2 and 3 your provided

Reply:
i changed the text in this readme but i can't find the readme for `easy-coding-standard-prefixed`. the repo itself is readonly and in the current repo, if i search for `easy-coding-standard-prefixed` or `Easy Coding Standard Prefixed`, i only find the file i edited.

Reply:
@TomasVotruba where to find the `README.md` of `easy-coding-standard-prefixed`?

Reply:
It's standalone package: https://github.com/symplify/easy-coding-standard-prefixed

Reply:
:confused: i can swear that i read the repo is read-only... whatever. thanks xD